DirectProgramming/Fortran/DenseLinearAlgebra/optimize-integral/README.md

```diff
@@ -0,0 +1,194 @@
+# `Optimization Integral`
```

Comment:
Since you mention timings in the README.md, do you need that performance disclaimer?  I’m thinking like the disclaimer we add to presentations and all the IDZ articles

Reply:
Not needed, no.  we discussed in email with Paul Petersen